pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -633,6 +641,35 @@ private void UpdateListPeptide(int selectPeptideIndex)
             }
         }
 
+
+        public class comboOption
+        {
+            public SpectrumInfoLibrary library;

```

Comment:
This field should be called "spectrumInfo", or maybe "spectrumInfoLibrary". 

A "SpectrumInfoLibrary" is a type of "SpectrumInfo", not a type of "Library".